pulp_rpm/app/serializers.py

```diff
@@ -548,21 +548,11 @@ class AddonSerializer(serializers.ModelSerializer):
     packages = serializers.CharField(
         help_text=_("Relative path to directory with binary RPMs.")
     )
-    repository = serializers.HyperlinkedRelatedField(
```

Comment:
Why do you remove repository from here? Is it because of plugin_managed repositories which are not  shown now?

Reply:
yes! I was using these tests to test https://github.com/pulp/pulpcore/pull/315

Reply:
moved to another PR: https://github.com/pulp/pulp_rpm/pull/1472